esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -2650,10 +2643,10 @@ void CWsDfuEx::getAPageOfSortedLogicalFile(IEspContext &context, IUserDescriptor
         wuTo.appendf("%4d-%02d-%02d %02d:%02d:%02d",year,month,day,hour,minute,second);
     }
 
-    char sortBy[256];
+    StringBuffer sortBy;
```

Comment:
I wondered about a stringAttr, but it is probably less efficient - since the StringBuffer contains a local buffer for small strings